tests/test_basic_s3.py

```diff
@@ -174,14 +174,28 @@ async def test_result_key_iters(s3_client, bucket_name, create_object):
     assert 'CommonPrefixes' in response
 
 
-async def test_can_get_and_put_object(s3_client, create_object, bucket_name):
-    await create_object('foobarbaz', body='body contents')
-    resp = await s3_client.get_object(Bucket=bucket_name, Key='foobarbaz')
+async def test_can_get_and_put_object(
+    s3_client, create_object, bucket_name: str
+):
+    key_name = 'foobarbaz'
+    await create_object(key_name, body='body contents')
+
+    resp = await s3_client.get_object(Bucket=bucket_name, Key=key_name)
     data = await resp['Body'].read()
     # TODO: think about better api and make behavior like in aiohttp
     resp['Body'].close()
     assert data == b'body contents'
 
+    # now test checksum'd file
+    key_name = 'foobarbaz_cs'
+    await create_object(key_name, b'abcd', ChecksumAlgorithm="sha1")
+    resp = await s3_client.get_object(
+        Bucket=bucket_name, Key=key_name, ChecksumMode="ENABLED"
+    )
+    data = await resp['Body'].read()
+    assert data == b'abcd'
+    print()
```

Comment:
Probably left over and should be removed?

Reply:
ah ya, from original commit from way back, thanks, removed